apps/preview/next/next.config.mjs

```diff
@@ -19,7 +19,7 @@ export default {
   webpack(config) {
     if (!isProd) {
       config.module.rules.push({
-        test: /\/sfui\/frameworks\/react\/index\.ts/,
+        test: /[\\\/]sfui[\\\/]frameworks[\\\/]react[\\\/]index\.ts/,
```

Comment:
please rebase now your branch with v2-develop since this fix is already merged. Let me know if you need help with this